src/components/footer/footer-link.tsx

```diff
@@ -1,20 +1,22 @@
 'use client'
 
 import Link from 'next/link'
+import { HTMLAttributeAnchorTarget } from 'react'
 
 import { useAnalytics } from '@/lib/hooks/use-analytics'
 
 type Props = {
   children: string
   href: string
+  target?: HTMLAttributeAnchorTarget | undefined
 }
 
-export function FooterLink({ children, href }: Props) {
+export function FooterLink({ children, href, target }: Props) {
   const { logEvent } = useAnalytics()
   return (
     <li>
       <Link
-        target='_blank'
+        target={target ?? '_blank'}
```

Comment:
I'm just wondering, any reason you wouldn't want a target _blank?

Reply:
Added for this case since `/legacy` is an _internal_ link